dashboard/src/main/java/org/openedx/dashboard/presentation/DashboardFragment.kt

```diff
@@ -445,6 +456,42 @@ private fun CourseItem(
                         maxLines = 2,
                         overflow = TextOverflow.Ellipsis
                     )
+                    Spacer(modifier = Modifier.height(4.dp))
+                    if (enrolledCourse.course.number.isNotEmpty()) {
+                        Row(
+                            Modifier
+                                .fillMaxWidth()
+                                .background(MaterialTheme.appColors.background),
+                            verticalAlignment = Alignment.CenterVertically,
+                        ) {
+                            Text(
+                                modifier = Modifier.testTag("txt_course_number"),
+                                text = enrolledCourse.course.number,
+                                color = MaterialTheme.appColors.textFieldHint,
+                                style = MaterialTheme.appTypography.labelMedium
+                            )
+                        }
+                        Spacer(modifier = Modifier.height(4.dp))
+                    }
```

Comment:
Instead of a row just for the course number, it can be added to the org text, the same as the web app.
![image](https://github.com/user-attachments/assets/0f1f90ce-a379-42c3-bfce-ca6a0db6ef3d)

```diff
--- a/dashboard/src/main/java/org/openedx/dashboard/presentation/DashboardFragment.kt
+++ b/dashboard/src/main/java/org/openedx/dashboard/presentation/DashboardFragment.kt
@@ -437,7 +437,7 @@ private fun CourseItem(
             ) {
                 Text(
                     modifier = Modifier.testTag("txt_course_org"),
-                    text = enrolledCourse.course.org,
+                    text = enrolledCourse.course.org+"+"+enrolledCourse.course.number,
                     color = MaterialTheme.appColors.textFieldHint,
                     style = MaterialTheme.appTypography.labelMedium
                 )
```

```suggestion
```